ui/app/AppLayouts/ActivityCenter/controls/NotificationHeaderRow.qml

```diff
@@ -0,0 +1,162 @@
+// NotificationHeaderRow.qml
+// -----------------------------------------------------------------------------
+// A compact header line for notification cards.
+// - Title (single line, truncated with "…")
+// - Optional verification badges (contact + trust indicator)
+// - Chat key shortened as "AAAA...ZZZZ"
+// - Unread indicator dot aligned to the far right
+//
+// USAGE EXAMPLES
+// --------------
+// // Basic usage with title and key
+// NotificationHeaderRow {
+//     title: "Alice Johnson"
+//     chatKey: "0x1234abcd5678efgh9012ijkl"
+// }
+//
+// // With badges
+// NotificationHeaderRow {
+//     title: "Community Bot"
+//     chatKey: "0x8765dcba4321"
+//     isContact: true
+//     trustedIndicator: 2
+// }
+//
+// // Without unread dot
+// NotificationHeaderRow {
+//     title: "Charlie"
+//     chatKey: "0xabcdef0987"
+//     unread: false
+// }
+//
+// NOTES
+// -----
+// * Maximum visible characters for `title` is controlled by maxNameChars.
+// * The unread dot uses Theme.primaryColor1 and sits at the far right.
+// * Badges are placed inline between the display name and the chat key.
+// * Fonts, spacing, and colors follow Theme definitions.
+// -----------------------------------------------------------------------------
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import StatusQ.Core
+import StatusQ.Core.Theme
+import StatusQ.Components
+
+import StatusQ.Core.Utils
+
+Control {
+    id: root
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // API
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // Titile (truncated to maxTitleChars)
+    required property string title
+
+    // Chat key (formatted as prefix…suffix)
+    property string chatKey: ""
+
+    // Show "is contact" badge if true
+    property bool   isContact: false
+
+    // Trust level indicator (0 = none). Expected values: StatusContactVerificationIcons.TrustedType
+    property int    trustedIndicator: 0
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Design parameters
```

Comment:
what about hiding those design-related values in `d`? It prevents users from using ad-hoc variants not matching the design.